preview/arcus-general.md

```diff
@@ -0,0 +1,71 @@
+# Arcus - Azure development in a breeze.
+
+## What is Arcus?
+A set of Codit-developed OSS libraries/tools that simplify building apps running on Microsoft Azure.
+It is Codit’s strategy to centralize company-wide best practices and use it across all our customers.
+
+* Built By Experts, For Experts
+  Sponsored by Codit! Maintained by a dedicated set of Codit experts that assures quality for our current and future clients.
+
+* Modular Design
+  All Arcus components are highly functional and composable. They can be used separately, but they are better together.
+
+* Open-Source & Free to use
+  The Arcus framework is constantly updated by contributions from Codit employees and abroad!
+  It is free to use under the MIT license.
+
+🚩 Are you doing the same thing at multiple customers? Then we are happy to hear! [Link to arcus GitHub]()
```

Comment:
What link should be put here ?

Reply:
Well, it would be nice if we have a form to link to or something? Whether this form is on GitHub, that would probably be difficult.

Reply:
And what should that form be about actually ?  I mean, what kind of questions / survey ?
It's not very clear to me what we want to achieve with that form.

Reply:
We previously had some sort of a questionnaire to ask people about what components they are using, their experience with those components, and/or ideas/suggestions or something. If we get a 'go' from management on the 'customer approval', we can also ask them on which customer projects they use Arcus.